cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -15,12 +15,31 @@ MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
 SHARED_DIRECTORY_COMPUTE="<%= @shared_directory_compute %>"
 SHARED_DIRECTORY_LOGIN="<%= @shared_directory_login %>"
+CONFIG_PROCESS="False"
+
+while getopts "c:" opt; do
+  case $opt in
+    c)
+      if [[ $OPTARG != "True" && $OPTARG != "False" ]]; then
+        echo "Invalid value for -c: $OPTARG. Expected 'True' or 'False'."
+        exit 1
+      fi
+      CONFIG_PROCESS=$OPTARG
+      ;;
+    *)
+      echo "Usage: $0 [-c True/False]" >&2
+      exit 1
+      ;;
+  esac
+done
```

Comment:
You can get the same behavior with a simple flag, without having to pass either `True` or `False` to the argument:

```bash
CONFIG_PROCESS=false
while getopts "c" opt; do
  case $opt in
    c) CONFIG_PROCESS=true;;
    ?) # echo usage output and exit
  esac
done
```

Reply:
Thanks for reminding, Jacopo. Done!